src/engagement_db_coda_sync/sync_stats.py

```diff
@@ -0,0 +1,41 @@
+from core_data_modules.logging import Logger
+
+log = Logger(__name__)
+
+
+class CodaSyncEvents:
+    READ_MESSAGE = "read_message"
+    SET_CODA_ID = "set_coda_id"
+    ADD_TO_CODA = "add_to_coda"
```

Comment:
From L7 I think this also can be `ADD_MESSAGE`

Reply:
Changed to ADD_MESSAGE_TO_CODA and READ_MESSAGE_FROM_ENGAGEMENT_DB to make these super clear